### PR TITLE
Add links "plugins" & "templates" to intro_search.txt

### PR DESCRIPTION
**Pull request by chang-zhao**
As suggested in https://github.com/splitbrain/dokuwiki/issues/2255#issuecomment-365910357 it will help wiki admins find useful plugins & templates

### CODE REVIEW

File: lib/plugins/extension/lang/en/intro_search.txt [hunk 1]
```diff
@@ -1 +1 @@
-This tab gives you access to all available 3rd party plugins and templates for DokuWiki. Please be aware that installing 3rd party code may pose a **security risk**, you may want to read about [[doku>security#plugin_security|plugin security]] first.
\ No newline at end of file
+This tab gives you access to all available 3rd party [[doku>plugins]] and [[doku>template]]s for DokuWiki. Please be aware that installing 3rd party code may pose a **security risk**, you may want to read about [[doku>security#plugin_security|plugin security]] first.
```
Comment (phy25) [inline lib/plugins/extension/lang/en/intro_search.txt]:
```suggestion
This tab gives you access to all available 3rd party [[doku>plugins|plugins]] and [[doku>template|templates]] for DokuWiki. Please be aware that installing 3rd party code may pose a **security risk**, you may want to read about [[doku>security#plugin_security|plugin security]] first.
```

Comment (chang-zhao) [reply lib/plugins/extension/lang/en/intro_search.txt]: Thank you. Indeed, I can translate to farsi, zhou etc.! I feel all those countries and people so closer... :) Updated.

**Comment by chang-zhao** [comment]
Updated & corrected with all existing translations of the file.

**Comment by phy25** [comment]
Wow, thanks for your efforts! I have squash-merged your branch at b520c36b0c66a11631d34bd24d5bc7b51a4cf85c. 🚀

**Pull request closed without merging**